### PR TITLE
loosen SCC capabilities check to allow stricter caps

### PR DESCRIPTION
**Pull request by stlaz**
needed for https://github.com/openshift/cluster-kube-apiserver-operator/pull/1338

/assign @s-urbaniak

### CODE REVIEW

File: test/extended/security/scc.go [hunk 1]
```diff
@@ -270,7 +270,10 @@ var _ = g.Describe("[sig-auth][Feature:SecurityContextConstraints] ", func() {
 		pod, err := exutil.NewPodExecutor(oc, "restrictedcapsh", image.ShellImage())
 		o.Expect(err).NotTo(o.HaveOccurred())
 
+		// TODO: remove desiredCapabilities once restricted-v2 is the default
+		// system:authenticated SCC in the cluster - in favour of alternativeDesiredCapabilities
 		desiredCapabilities := "000000000000051b"
```
Comment (s-urbaniak) [inline test/extended/security/scc.go:275]: note to myself (also maybe worth a comment?): this corresponds to `cap_chown,cap_dac_override,cap_fowner,cap_fsetid,cap_setpcap,cap_net_bind_service`

File: test/extended/security/scc.go [hunk 2]
```diff
@@ -270,7 +270,10 @@ var _ = g.Describe("[sig-auth][Feature:SecurityContextConstraints] ", func() {
...
+		alternativeDesiredCapabilities := "0000000000000000"
```
Comment (s-urbaniak) [inline test/extended/security/scc.go:276]: suggestion to make it explicit: `capsNone`

**Comment by s-urbaniak** [comment]
(comment can be applied in the TODO follow-up)

File: test/extended/security/scc.go (hunk 1 above)
Comment (mfojtik) [reply test/extended/security/scc.go:275]: yes please :-)

Comment (stlaz) [reply test/extended/security/scc.go:275]: these caps are going to be dropped once the related PR merges, I don't see much value describing it during its final days as it has lived undescribed for what I'm going to guess is more than 5 years

**Pull request merged**